sdk/core/azure-core-jackson/src/main/java/com/azure/android/core/serde/jackson/JacksonSerder.java

```diff
@@ -40,7 +40,11 @@
  */
 public final class JacksonSerder {
     private static final Pattern PATTERN = Pattern.compile("^\"*|\"*$");
-    private static JacksonSerder jacksonSerder;
+
+    private static final int CACHE_SIZE_LIMIT = 10000;
```

Comment:
Based on @anuchandy's answer [here](https://github.com/Azure/azure-sdk-for-android/pull/972/files#r868513723), I think this value looks good but we should also pay close attention to it.